zio-http/src/main/scala/zhttp/service/server/ServerChannelInitializer.scala

```diff
@@ -47,7 +47,7 @@ final case class ServerChannelInitializer[R](
 
     // ObjectAggregator
     // Always add ObjectAggregator
-    pipeline.addLast(HTTP_OBJECT_AGGREGATOR, new HttpObjectAggregator(cfg.maxRequestSize))
+    // pipeline.addLast(HTTP_OBJECT_AGGREGATOR, new HttpObjectAggregator(cfg.maxRequestSize))
```

Comment:
This should be conditionally added via server config. Should be true by default.